README.md

```diff
@@ -1,15 +1,59 @@
 # hello-hydra
 
-An example eosio.token contract tested with [Hydra](docs.klevoya.com/hydra).
+[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)
 
-To install the dependencies run:
+[![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+Hydra is an EOSIO smart contract testing framework that allows you to quickly get started with testing your smart contract code. Besides Hydra, installing additional software or running your own local blockchain node is not required. 
+
```

Comment:
Do we need dillinger in this repo? As this is an example repo for new Hydra users, I'd say we should keep the dependencies as lean as possible to not overload the users with additional non-Hydra related information